lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -864,6 +871,48 @@ mkUnsignedTx era ttl cs md wdrls certs =
         toErrMkTx :: Cardano.TxBodyError AllegraEra -> ErrMkTx
         toErrMkTx = ErrConstructedInvalidTx . T.pack . Cardano.displayError
 
+
+    --mkMaryTx :: Either ErrMkTx (Cardano.TxBody MaryEra)
+    mkMaryTx = left toErrMkTx $ Cardano.makeTransactionBody $ Cardano.TxBodyContent
+        { Cardano.txIns =
+            toCardanoTxIn . fst <$> CS.inputs cs
+
+        , Cardano.txOuts =
+            toMaryTxOut <$> CS.outputs cs
+
+        , Cardano.txWithdrawals =
+            Cardano.TxWithdrawals Cardano.WithdrawalsInMaryEra wdrls
+
+        , Cardano.txCertificates =
+            Cardano.TxCertificates Cardano.CertificatesInMaryEra certs
+
+        , Cardano.txFee =
+            Cardano.TxFeeExplicit Cardano.TxFeesExplicitInMaryEra fees
+
+        , Cardano.txValidityRange =
+            ( Cardano.TxValidityNoLowerBound
+            , Cardano.TxValidityUpperBound Cardano.ValidityUpperBoundInMaryEra ttl
+            )
+
+        , Cardano.txMetadata =
+            maybe
+                Cardano.TxMetadataNone
+                (Cardano.TxMetadataInEra Cardano.TxMetadataInMaryEra)
+                md
+
+        , Cardano.txAuxScripts =
+            Cardano.TxAuxScriptsNone
+
+        , Cardano.txUpdateProposal =
+            Cardano.TxUpdateProposalNone
+
+        , Cardano.txMintValue =
+            Cardano.TxMintNone
+        }
+      where
+        --toErrMkTx :: Cardano.TxBodyError MaryEra -> ErrMkTx
```

Comment:
Why is this commented out ?

Reply:
If I try to import `MaryEra` like the other eras I get:

```
/Users/anviking/IOHK/cardano-wallet/lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs:85:7: error:
    In module ‘Cardano.Wallet.Shelley.Compatibility’:
      ‘MaryEra’ is a data constructor of ‘CardanoEra’
    To import it use
      import Cardano.Wallet.Shelley.Compatibility( CardanoEra( MaryEra ) )
    or
      import Cardano.Wallet.Shelley.Compatibility( CardanoEra(..) )
   |
85 |     , MaryEra
   |       ^^^^^^^
```

despite other eras being importable just fine:
```
    ( AllegraEra
    , MaryEra
    , ShelleyEra
```
If I switch to importing `CardanoEra (MaryEra)`, then I get this strange error

```
/Users/anviking/IOHK/cardano-wallet/lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs:911:42: error:
    • Expected a type, but
      ‘MaryEra’ has kind
      ‘CardanoEra Cardano.MaryEra’
    • In the first argument of ‘Cardano.TxBodyError’, namely ‘MaryEra’
      In the type signature:
        toErrMkTx :: Cardano.TxBodyError MaryEra -> ErrMkTx
      In an equation for ‘mkMaryTx’:
          mkMaryTx
            = left toErrMkTx
                $ Cardano.makeTransactionBody
                    $ Cardano.TxBodyContent
                        {Cardano.txIns = toCardanoTxIn . fst <$> CS.inputs cs,
                         Cardano.txOuts = toMaryTxOut <$> CS.outputs cs,
                         Cardano.txWithdrawals = Cardano.TxWithdrawals
                                                   Cardano.WithdrawalsInMaryEra wdrls,
                         Cardano.txCertificates = Cardano.TxCertificates
                                                    Cardano.CertificatesInMaryEra certs,
                         Cardano.txFee = Cardano.TxFeeExplicit
                                           Cardano.TxFeesExplicitInMaryEra fees,
                         Cardano.txValidityRange = (Cardano.TxValidityNoLowerBound,
                                                    Cardano.TxValidityUpperBound
                                                      Cardano.ValidityUpperBoundInMaryEra ttl),
                         Cardano.txMetadata = maybe
                                                Cardano.TxMetadataNone
                                                (Cardano.TxMetadataInEra
                                                   Cardano.TxMetadataInMaryEra)
                                                md,
                         Cardano.txAuxScripts = Cardano.TxAuxScriptsNone,
                         Cardano.txUpdateProposal = Cardano.TxUpdateProposalNone,
                         Cardano.txMintValue = Cardano.TxMintNone}
            where
                toErrMkTx :: Cardano.TxBodyError MaryEra -> ErrMkTx
                toErrMkTx = ErrConstructedInvalidTx . T.pack . Cardano.displayError
    |
911 |         toErrMkTx :: Cardano.TxBodyError MaryEra -> ErrMkTx
    |                                          ^^^^^^^
```

Can remove the comments completely for now.